apps/i18n/common/en_us.json

```diff
@@ -586,13 +586,17 @@
   "projectGalleryCardAction": "See the gallery",
   "projectGalleryCardDescription": "Share your own creations or find inspiration from creations made by students all around the world.",
   "projectGroupArtist": "Drawing",
+  "projectGroupArtistAllProjects": "All Drawing Projects",
   "projectGroupArtistViewMore": "View more Drawing projects",
   "projectGroupEvents": "Games with Events",
+  "projectGroupEventsAllProjects": "All Games with Events Projects",
   "projectGroupEventsViewMore": "View more Games with Events projects",
   "projectGroupMinecraft": "Minecraft",
+  "projectGroupMinecraftAllProjects": "All Minecraft Projects",
   "projectGroupMinecraftViewMore": "View more Minecraft projects",
   "projectGroupPlaylab": "Stories and Games with Play Lab",
-  "projectGroupPlaylabViewMore": "View more Play Lab projects",
+  "projectGroupPlaylabAllProjects": "All Stories and Games with Play Lab Projects",
+  "projectGroupPlaylabViewMore": "View more Stories and Games with Play Lab projects",
```

Comment:
These are very long and will likely be longer is other languages, can we get away with just "View more Play Lab projects" or is this string required?

Reply:
@poorvasingal , what do you think about these strings? you can see them in context in the anigif